src/data/cakeConfig.json

```diff
@@ -54,63 +54,72 @@
           "image": "/assets/Corpus.png",
           "imgName": "Korpus",
           "heading": "Korpus",
-          "options": ["Vanilka", "Kakao", "Bezlepkový orechový"]
+          "options": ["Vanilka", "Kakao"],
+          "text": []
         },
         {
           "image": "/assets/Cream.png",
           "imgName": "Krem",
           "heading": "Krém",
-          "options": [
-            "Mascarpone s tvarohom a redukciou z lesného ovocia",
-            "Mascarpone s kokosom, malinovou redukciou a čerstvými malinami ",
-            "Čokoláda a slaný karamel s chrumkavými kúskami karamelu v čokoláde a pomarančovou redukciou",
-            "Čokoláda s maracujou",
-            "Maková s mascarpone a višňovou redukciou",
-            "Ruby čokoláda s mascarpone a čerstvými malinami (plus 5€, ruby je nová prírodne ružová a kyselkavá odroda čokolády spoločnosti callebaut)",
-            "Mascarpone s tvarohom a limoncellom"
+          "options": [],
+          "text": [
+            {
+              "option": "Mascarpone a lesné ovocie",
+              "description": "Veľmi svieži krém z mascarpone s jemným tvarohom, korpusy nasiaknuté redukciou z lesného ovocia a čerstvé bobuľové ovocie"
+            },
+            {
+              "option": "Čokoláda a maliny",
+              "description": "Ten NAJ jčokoládovejší buttercream osviežený čerstvými malinami a korpusy nasiaknuté redukciou z lesného ovocia"
+            },{
+              "option": "Tvaroh, lemon curd a čučoriedky",
+              "description": "Svieža kyselkavá chuť tvarohu a jemného mascarpone doplnená našou krémovou citrónovou plnkou s čerstvými čučoriedkami.Odporúčame s čokoládovým korpusom"
+            },{
+              "option": "Mak a višňa",
+              "description": "Nesmrteľná a žiadúca kombinácia maku a višní v kréme z mascarpone Na Vaše želanie aj kombinácia MAK a LEMON CURD"
+            },{
+              "option": "Čokoláda a slaný karamel",
+              "description": "Čokoládový buttercream a lahodný domáci slaný karamel"
+            }
           ]
         },
         {
           "image": "/assets/Design.png",
           "imgName": "VonkajsiePrevedenie",
           "heading": "Vonkajšie prevedenie",
-          "options": ["Polonahá torta", "Celá obtretá", "Obtretá na hladko", "Obtretá nedbalo"]
+          "options": ["Polonahá torta jemne obtretá s krémom ktorým je aj naplnená, korpusy presvitajú a vytvárajú naturálny finish", "Obtretá s buttercremom na hladko pre elegantný finish", "Obtretá s buttercreamom nedbalo pre rustikálny finish", "Čokoládová ganache (pre dospelých aj so štipkou alkoholu) ideálna pre špecifické minimalistické zdobenia jedlou farbou alebo prachom", "Fondant (cena na vyžiadanie, odporúčame len pri svadobných tortách a náročných ručne maľovaných tortách na mieru)"],
```

Comment:
Polonahá jemne obtretá s krémom ktorým je aj naplnená

huehuehueh

infantilne joky aside, maju tam kopec gramatickych chyb, ale asi jebat na nich

Reply:
Bohvie čo si predstavovali keď to vymýšľali :D